### PR TITLE
implement Eu Indicators Tab

### PR DESCRIPTION
**Pull request by alexef**

### CODE REVIEW

File: src/nbsap/templates/eu_indicators.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
+{% extends 'layout.html' %}
+{% load i18n %}
+{% load utils %}
+
+
+{% block title %}
+  NBSAP - {% trans "EU Indicators" %}
```
Comment (alexef) [inline src/nbsap/templates/eu_indicators.html]: Are you sure about the "NBSAP" prefix?

File: src/nbsap/views/indicators.py [hunk 2]
```diff
@@ -23,3 +23,15 @@ def indicator(request, pk):
     return render(request, 'indicators/indicator_details.html', {
         'indicator': indicator,
     })
+
+
+def list_indicators(request, pk=None):
+    if not pk:
+        indicator = models.EuIndicator.objects.all()[0]
```
Comment (alexef) [inline src/nbsap/views/indicators.py]: We should use first(), but that's django 1.6 and we're stuck with 1.5 here, right?

Comment (alexef) [inline src/nbsap/views/indicators.py]: Also, please crash silently if there are no indicators in the db.

Comment (androane) [inline src/nbsap/views/indicators.py]: Yes, there's no "first" in 1.5

**Comment by alexef** [comment]
Lgtm, ok to merge!

**Pull request merged**